client/src/Pages/User/Cart/CartProducts.js

```diff
@@ -28,12 +28,17 @@ const CartProduct = ({ product, deleteItem }) => {
   const handleMouseOut = () => {
     setIsMouseOver(false)
   }
+
   const increase = () => {
     setQuantity(quantity + 1)
+    updateQuantity(quantity + 1)
+    deleteItem(date.getTime())
```

Comment:
sorry why are we deleting the item here? I see you created a dummyDelete function, but why would we need to delete an item if we adjust the quantity?

Reply:
Oh that's just an alias for dummyDelete. Sorry the dummyDelete function was written by someone else and I just reused it here, the name does not have a good semantics. I just changed the function name here. All it does is to force update the parent element.